.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ["2.5", "2.6", "2.7", "3.0", "3.1", ruby-head, jruby-9.2, jruby-9.3]
+        rubocop_version: ["0.86", "1.20"]
+        exclude:
+          - ruby: "2.4"
+            rubocop_version: "1.20"
```

Comment:
These lines shouldn’t be necessary, since Ruby 2.4 is not in the matrix at all.

Reply:
@bquorning Removed.